### ISSUE TITLE
New setting - Teams

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#69]
By Saus and Me
This would be a setting/category in the tournament section. Teams would consist to up to 4 players, which would work together to gather up the points needed or most points to win.
Settings include
Number of teams (1 - 4 teams, Blue, Red, Green, and Yellow)

**Comment by EmeraldLoc** [comment]
Sounds like a good feature for 2.5

**Comment by Bear64DX** [comment]
It'd also be cool if you could name them as well (with hex codes if you want).

**Comment by TheMan6900** [comment]
yeah exactly bear "Allow players to name there team there on. (basically the same way as tourney naming). Maybe players could vote on what name they want but not sure"

**Comment by TheMan6900** [comment]
Oh yeah team chat would be cool

**Comment by Bear64DX** [comment]
Freeze Tag could have a new gimmick where it's a team FFA. Everyone would be a tagger and would be assigned to a team. A team tries to freeze every opponent on the opposite team.

**Comment by Bear64DX** [comment]
Royale would also have a team mode.
